### PR TITLE
Use STANDARD storage class to mark non-cached checkout objects

### PR DESCRIPTION
**Pull request by xbrianh**
addresses #2145

### CODE REVIEW

File: infra/buckets/gs.tf [hunk 1]
```diff
@@ -41,7 +41,7 @@ resource google_storage_bucket dss_gs_checkout_bucket {
     }
     condition {
       age = "${var.DSS_BLOB_TTL_DAYS}"
-      matches_storage_class = ["DURABLE_REDUCED_AVAILABILITY"]
+      matches_storage_class = ["STANDARD"]
```
Comment (amarjandu) [inline infra/buckets/gs.tf]: can we change this for the test buckets too?

Comment (xbrianh) [reply infra/buckets/gs.tf]: The test bucket lifecycle rule is not based on storage class, so it doesn't need to be updated.

**Comment by xbrianh** [comment]
@DailyDreaming Thanks for pointing those out. Fixed.

File: dss/stepfunctions/gscopyclient/implementation.py [hunk 2]
```diff
@@ -69,10 +69,10 @@ def run(self) -> dict:
             if not will_cache:
                 logger.info("Not caching %s with content-type %s size %s", self.source_key, content_type, self.size)
 
-            # TODO: DURABLE_REDUCED_AVAILABILITY is being phased out by Google; use a different method in the future
+            # TODO: STANDARD is being phased out by Google; use a different method in the future
```
Comment (DailyDreaming) [inline dss/stepfunctions/gscopyclient/implementation.py]: This TODO can be deleted now.

Comment (xbrianh) [reply dss/stepfunctions/gscopyclient/implementation.py]: will fix

File: tests/test_checkout_caching.py [hunk 3]
```diff
@@ -137,13 +137,13 @@ def test_google_cached_checkout_creates_multiregional_storage_type(self, mock_ch
     def test_google_uncached_checkout_creates_durable_storage_type(self, mock_check):
```
Comment (mikebaumann) [inline tests/test_checkout_caching.py]: nit: You could also update the test name, perhaps:
```
test_google_uncached_checkout_creates_uncached_storage_type
```

Comment (xbrianh) [reply tests/test_checkout_caching.py]: fixed

**Comment by xbrianh** [comment]
@mikebaumann Performance testing is WIP, and should be handled as part of #2081 and #2082, although it would be useful, I think, to create a checklist of DSS operations that should undergo regular performance testing.

**Pull request merged**